pyhelm3/models.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, _command: Command, **kwargs):
 
 
 #: Type for a name (chart or release)
-Name = constr(pattern = r"^[a-z0-9-]+$")
+Name = constr(pattern = r"^[a-zA-Z0-9-]+$")
```

Comment:
I think the problem we have is this type gets re-used in a few too many places. Its based on this documentation:
https://helm.sh/docs/chart_best_practices/conventions/#chart-names

I think we need to rework this one, potentially.